src/shared/themed-bootstrap.css

```diff
@@ -12616,7 +12616,7 @@ strong {
   font-weight: 700;
 }
 
-@media (max-width: 991.98px) {
+<<<<<<< HEAD ======= @media (max-width: 991.98px) {
```

Comment:
@sreidthomas this is a remnant of a merge conflict. It needs to be fixed before we can merge in this PR.

Reply:
You should be able to fix it by re-running the `compile_styles.sh` script.

Reply:
I ran the script again and it keeps failing @maxatdetroit. I tried to switch the command I found in teams but it's still failing. Its not switching rem to em in that file

<img width="785" alt="Screenshot 2024-04-22 at 12 57 09 PM" src="https://github.com/CityOfDetroit/COD-Design-System/assets/141044304/433c7dc6-a749-44ae-b616-ccd03ea3cb34">

Reply:
<img width="787" alt="Screenshot 2024-04-22 at 1 02 14 PM" src="https://github.com/CityOfDetroit/COD-Design-System/assets/141044304/a576735a-d028-48f3-b02e-432085bc1f80">

Reply:
Try running `yarn install` first. <--- this is always a good step to make sure you have all the right dependencies installed.